.cfignore

```diff
@@ -2,8 +2,10 @@
 .github
 .idea
 acceptance-tests
-docs
 integration-tests
+terraform-tests
+service-images
```

Comment:
If we don't cf-push the service-images, how are we going to return the file in the catalog for appsman to display?